registry/api/errcode/errors_test.go

```diff
@@ -1,13 +1,14 @@
-package v2
+package errcode
 
 import (
-	"encoding/json"
```

Comment:
Please delete. Are you using goimports with your editor? This becomes really easy using that tool.